mne/io/edf/edf.py

```diff
@@ -392,6 +391,7 @@ def _get_edf_info(fname, stim_channel, annot, annotmap, eog, misc, exclude,
                  ' Inferring from the file size.')
             edf_info['n_records'] = n_records = read_records
 
+    n_samps = n_samps[include]
```

Comment:
moving this didn't change anything functionally, right? I don't see `n_samps` or `include` modified after it was used above